src/coreclr/src/tools/crossgen2/ILCompiler.Reflection.ReadyToRun/ILCompiler.Reflection.ReadyToRun.csproj

```diff
@@ -7,19 +7,14 @@
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <AssemblyKey>Open</AssemblyKey>
     <IsDotNetFrameworkProductAssembly>true</IsDotNetFrameworkProductAssembly>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
```

Comment:
@cshung Was there a reason why you made this netstandard2.0 and not `$(NetCoreAppCurrent)`?

Reply:
@jkotas, thanks for bringing this to my attention.

The `ILCompiler.Reflection.ReadyToRun` library is currently used by [ILSpy](https://github.com/icsharpcode/ILSpy), a C# decompiler.

As of now, it needs to run on both .NET framework and .NET Core. Therefore I think .NET Standard 2.0 is the best choice.

Right now, the actual binaries they use is built on my machine (so I get to control what bits get to ILSpy) and published on my personal feed. 

[Here](https://github.com/icsharpcode/ILSpy/blob/e0ee8bf286f23315ad7345f83f460e7f11741df4/ILSpy.ReadyToRun/ILSpy.ReadyToRun.csproj#L60) is where they are consuming the binary right now through [my feed](https://github.com/icsharpcode/ILSpy/blob/e0ee8bf286f23315ad7345f83f460e7f11741df4/NuGet.config#L7).

If it is necessary, I can still built against .NET Standard 2.0 and publish them to my own feed.

Long term, I would love to have them consuming the binary from a feed that the official build publishes. That said, I have no plan to do this now.

Reply:
cc @ericstj